tests/framework/config/cluster.go

```diff
@@ -27,8 +27,9 @@ const (
 )
 
 type ClusterConfig struct {
-	ClusterSize       int
-	PeerTLS           TLSConfig
-	ClientTLS         TLSConfig
-	QuotaBackendBytes int64
+	ClusterSize                int
+	PeerTLS                    TLSConfig
+	ClientTLS                  TLSConfig
+	QuotaBackendBytes          int64
+	DisableStrictReconfigCheck bool
```

Comment:
The name `DisableStrictReconfigCheck` seems to be anti-pattern because users (test cases) need to use double negative to enable it, `not disabled == enabled`. It's also a tradeoff, please see https://github.com/etcd-io/etcd/pull/14389#discussion_r955674249 .

You just follow the same naming as the existing e2e configuration `NoStrictReconfig`, so it isn't your fault. So we can keep it as it's for now, and feel free to change it to `StrictReconfigCheck` so as to be consistent with the definition in [the production code](https://github.com/etcd-io/etcd/blob/3c79a4dc3801b347a91b114524bdeb4712573a5e/server/embed/config.go#L240) in a separate PR.